src/bootstrap/lib.rs

```diff
@@ -1366,7 +1366,7 @@ impl Build {
                 eprintln!(
                     "
 Couldn't find required command: ninja
-You should install ninja, or set ninja=false in config.toml
+You should install ninja, or set ninja=false in config.toml under the [llvm] section.
```

Comment:
```suggestion
You should install ninja, or set `ninja=false` in config.toml in the `[llvm]` section.
```

This should either be "under the `[llvm]` section header" or "in the `[llvm]` section", and I think the latter is clearer.